clang/test/CheckedCRewriter/b11_calleestructnp.c

```diff
@@ -53,3 +72,9 @@ struct np *bar() {
   return z;
 }
 //struct np *bar() {
+//CHECK_NOALL: struct np *bar() {
+//CHECK_NOALL:   struct np *z = sus(x, y);
+//CHECK_NOALL: //struct np *bar() {
```

Comment:
Why is this `//struct np *bar()` ? Seems odd. Maybe we should just delete the commented-out code that's there now?